cmd/invoke.go

```diff
@@ -75,6 +86,7 @@ func init() {
 	InvokeCmd.Flags().StringVarP(&invokeVerb, "verb", "v", defaultHTTPVerb, "The HTTP verb to use")
 	InvokeCmd.Flags().StringVarP(&invokeDataFile, "data-file", "f", "", "A file containing the JSON serialized data (optional)")
 	InvokeCmd.Flags().BoolP("help", "h", false, "Print this help message")
+	InvokeCmd.Flags().BoolVar(&invokeUseSocket, "enable-domain-socket", false, "Invoke thought Unix domain socket, not supported on Windows")
```

Comment:
Set a short version too for this param. Maybe "u" and rename the long version to "unix-domain-socket"

Reply:
Also, the path might not always be "/tmp", since @yaron2 has plans to make it work in K8s - which requires a shared volume. These commands should assume "/tmp" is configurable here too.

Reply:
For now, since `dapr/dapr` only support `/tmp`, may update it first I think,

In my proposal, standalone mode support is first step, and the next is to support in K8s.

@yaron2 could share more about this plan here?

Reply:
@artursouza changed, please review again.

Reply:
The concern still remains here regarding this being a string and not a boolean param. The problem is that the CLI params behave like a public API, so we need to keep supporting it or deprecate later. To avoid rework here, we should make the path to be configurable in daprd and make this param a string instead of bool.

/cc @yaron2 @wcs1only

Reply:
To sync with runtime, I can make it as string and use tmp path as default value. I would do it after you agree.

Reply:
Please do that @daixiang0

Reply:
Please review @artursouza @yaron2

Reply:
Are the test failures related to the change?

Reply:
Since go-sdk still does not support it, only unit tests added.